configure.ac

```diff
@@ -203,6 +203,12 @@ AC_ARG_ENABLE([tcti-pcap],
             [enable_tcti_pcap=yes])
 AM_CONDITIONAL([ENABLE_TCTI_PCAP], [test "x$enable_tcti_pcap" != xno])
 
+AC_ARG_ENABLE([tcti-libtpms],
+            [AS_HELP_STRING([--disable-tcti-libtpms],
+                            [don't build the tcti-libtpms module])],,
+            [enable_tcti_libtpms=yes])
+AM_CONDITIONAL([ENABLE_TCTI_LIBTPMS], [test "x$enable_tcti_libtpms" != xno])
+
```

Comment:
I guess, libtpms does not have a .pc file ? In that case we should call AC_CHECK_LIB somewhere.

Reply:
My bad, we're not linking, so it should be AC_CHECK_HEADERS not AC_CHECK_LIB

Reply:
Added.